ReSharper.FSharp/test/data/features/daemon/identifierHighlighting/Extension members 02.fs.gold

```diff
@@ -0,0 +1,22 @@
+﻿open |System|(0)
+open |System|(1).Linq
+
+type |Char|(2) with
+    static member |IsZero|(3) |x|(4) = |x|(5) |=|(6) |char|(7) 0
+
+let |linq|(8) (|x|(9) : |string|(10)) =
+    |x|(11).Select(Char.IsZero).ToArray()
```

Comment:
How can I make this thing work? It skips Select and ToArray. Maybe add more references somewhere?